electron/legendary/user.ts

```diff
@@ -20,19 +21,29 @@ export class LegendaryUser {
     return new Promise((res) => {
       const child = spawn(legendaryBin, command)
       child.stderr.on('data', (data) => {
-        console.log(`stderr: ${data}`)
+        logInfo(`stderr: ${data}`)
         if (`${data}`.includes('ERROR')) {
+          logError(`${data}`)
+          dialog.showErrorBox(
+            i18next.t('box.error.login.tittle', 'Cannot Login!'),
+            `${data}`
+          )
           return res('error')
```

Comment:
I don't like the logInfo on top. If an error occurs data will be logged twice. Once as info and once as error. Just log one of them like so:

```suggestion
        if (`${data}`.includes('ERROR')) {
          logError(`${data}`, LogPrefix.Legendary)
          dialog.showErrorBox(
            i18next.t('box.error.login.tittle', 'Cannot Login!'),
            `${data}`
          )
          return res('error')
       } else {
          logInfo(`stderr: ${data}`, LogPrefix.Legendary)
       }
```